paper/paper.bib

```diff
@@ -0,0 +1,46 @@
+@article{tsoulis2012analytical,
+  title={Analytical computation of the full gravity tensor of a homogeneous arbitrarily shaped polyhedral source using line integrals},
+  author={Tsoulis, Dimitrios},
+  journal={Geophysics},
+  volume={77},
+  number={2},
+  pages={F1--F11},
+  year={2012},
+  publisher={Society of Exploration Geophysicists}
+}
+
+@article{tsoulis2021computational,
+  title={A computational review of the line integral analytical formulation of the polyhedral gravity signal},
+  author={Tsoulis, Dimitrios and Gavriilidou, Georgia},
+  journal={Geophysical Prospecting},
+  volume={69},
+  number={8-9},
+  pages={1745--1760},
+  year={2021},
+  publisher={European Association of Geoscientists \& Engineers}
+}
+
+@article{izzo2022geodesy,
+  title={Geodesy of irregular small bodies via neural density fields},
+  author={Izzo, Dario and G{\'o}mez, Pablo},
+  journal={Communications Engineering},
+  volume={1},
+  number={1},
+  pages={48},
+  year={2022},
+  publisher={Nature Publishing Group UK London}
+}
+
+@inproceedings{GeodesyNetsBenchmark,
+  title={Investigation of the Robustness of Neural Density Fields},
+  author={Jonas Schuhmacher and Fabio Gratl and Dario Izzo and Pablo G{\'o}mez},
+  booktitle={Proceedings of the 9th International Conference on Astrodynamics Tools and Techniques, ICATT},
+  year={2023},
+}
+
+@inproceedings{marak2023trajectory,
```

Comment:
Isn't it the other way around?
Rasmus' paper in
Proceedings of the 9th International Conference on Astrodynamics Tools and Techniques, ICATT
And mine in
Proceedings of the 12th International Conference on Guidance, Navigation \& Control Systems (GNC)